admin-ui/src/component/reservations/requested/RecurringReservationsView.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import styled from "styled-components";
+import {
+  Query,
+  QueryReservationByPkArgs,
+  type ReservationType,
+} from "common/types/gql-types";
+import { useTranslation } from "react-i18next";
+import { useQuery } from "@apollo/client";
+import { format } from "date-fns";
+import { RECURRING_RESERVATION_QUERY } from "./queries";
+import { useNotification } from "../../../context/NotificationContext";
+import ReservationList from "../../ReservationsList";
+
+const StyledHeading = styled.h3`
```

Comment:
For consistency, headings are defined in "common/src/common/typography"

Reply:
Yeah. It's not the same style, it might be semantically table header instead of h3 but the data is not in a table format but a list. The reason to use h3 over div here is just for aria.